src/libs/elektra/elektra.c

```diff
@@ -0,0 +1,88 @@
+/**
+ * @file
+ *
+ * @brief Elektra High Level API.
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#include "elektra.h"
+#include <stdlib.h>
+#include <memory.h>
+
+Elektra * elektraOpen (const char * application)
```

Comment:
Please add error handling here!

It is important to understand which guarantees we can give to libraries.